### PR TITLE
added Welcome, Stale Bot

### PR DESCRIPTION
**Pull request by anjali1102**

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: anjali1102/Greeting-action@main
```
Comment (HarshCasper) [inline .github/workflows/greetings.yml]: Do you really think it would work?

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          issue_message: 'Hello @${{github.actor}}, <br>
+            Thank you for opening an issue. :partying_face: <br>
+            To get assigned to this particular issue please use <b>/assign</b><br>
+            <i> Check this <a href="https://docs.google.com/document/d/1WYYc2ymImxS6-TKB6pRX77A9zpXb_ujGjdpfMaTDXwc/edit">guide</a> before contributing.'
```
Comment (HarshCasper) [inline .github/workflows/greetings.yml]: Don't copy-paste actions from other projects (In this case NeoAlgo) to other projects. It kills the open-source spirit.

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: >
+            Please reopen this issue once you add more information and updates here.
+            If this is not the case and you need some help, feel free to seek help
+            from our [Telegram](https://t.me/tesseractcoding) or ping one of the
```
Comment (HarshCasper) [inline .github/workflows/stale.yml]: Tesseract Coding? Really?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by anjali1102** [comment]
Review pls

**Review by HarshCasper — CHANGES_REQUESTED** [review]
We would still need a `.github/stale.yml` with a content similar to this: 
```
# Number of days of inactivity before an issue becomes stale
daysUntilStale: 7
# Number of days of inactivity before a stale issue is closed
daysUntilClose: 8
# Issues with these labels will never be considered stale
exemptLabels:
  - bug
  - do not merge yet
  - help wanted
  - high priority
  - pinned
# Label to use when marking an issue as stale
staleLabel: stale
# Comment to post when marking an issue as stale. Set to `false` to disable
markComment: >
  This issue has been automatically marked as stale because it has not had
  recent activity. It will be closed if no further activity occurs. Thank you
  for your contributions.
# Comment to post when closing a stale issue. Set to `false` to disable
closeComment: >
  Please reopen this issue once you commit the changes requested or
  make improvements on the code. Thank you for your contributions.
```

Example cited from NeoAlgo

**Comment by anjali1102** [comment]
> We would still need a `.github/stale.yml` with a content similar to this:
> 
> ```
> # Number of days of inactivity before an issue becomes stale
> daysUntilStale: 7
> # Number of days of inactivity before a stale issue is closed
> daysUntilClose: 8
> # Issues with these labels will never be considered stale
> exemptLabels:
>   - bug
>   - do not merge yet
>   - help wanted
>   - high priority
>   - pinned
> # Label to use when marking an issue as stale
> staleLabel: stale
> # Comment to post when marking an issue as stale. Set to `false` to disable
> markComment: >
>   This issue has been automatically marked as stale because it has not had
>   recent activity. It will be closed if no further activity occurs. Thank you
>   for your contributions.
> # Comment to post when closing a stale issue. Set to `false` to disable
> closeComment: >
>   Please reopen this issue once you commit the changes requested or
>   make improvements on the code. Thank you for your contributions.
> ```
> 
> Example cited from NeoAlgo

do I have to create new PR for this file/ delete this PR and again do?

**Pull request closed without merging**